src/services/pcn-synflood/swagger_api/synflood_swagger

```diff
@@ -759,9 +756,8 @@
                         "description": "OK: Successful operation",
                         "schema": {
                             "readOnly": true,
-                            "type": "integer",
-                            "description": "pkts requests to send over total received pkts",
-                            "format": "uint64"
+                            "type": "string",
+                            "description": "pkts requests to send over total received pkts"
```

Comment:
Ratio between the number of IP pkts requests to send by application protocols and the total number of received pkts